### PR TITLE
Change "attr-bans": "none" to "attr-bans": false in .htmllintrc (#43)

### LINKED ISSUE #43: HTML lint is reporting weird errors
If you know what this is or how to fix it, please let us know! 

I couldn't find bannedAttrs or for

```
[htmllint error in paste-templates/admin-created-account-welcome.html ] TypeError: bannedAttrs.forEach is not a function
[htmllint error in paste-templates/base.html ] TypeError: bannedAttrs.forEach is not a function
[htmllint error in paste-templates/digest.html ] TypeError: bannedAttrs.forEach is not a function
[htmllint error in paste-templates/email-confirmation.html ] TypeError: bannedAttrs.forEach is not a function
[htmllint error in paste-templates/password-reset_design_editor.html ] TypeError: bannedAttrs.forEach is not a function
[htmllint error in paste-templates/password-reset.html ] TypeError: bannedAttrs.forEach is not a function
[htmllint error in paste-templates/receipt.html ] TypeError: bannedAttrs.forEach is not a function
[htmllint error in paste-templates/welcome.html ] TypeError: bannedAttrs.forEach is not a function
```

### PR DESCRIPTION
**Pull request by i-ron-y**
Closes #43

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/email-templates?pullRequest=45) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/email-templates?pullRequest=45) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/email-templates?pullRequest=45) it.</sub>

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/email-templates?pullRequest=45) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/email-templates?pullRequest=45) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/email-templates?pullRequest=45) it.</sub>

**Comment by mbernier** [comment]
Ohh awesome! Thank you!!

**Comment by thinkingserious** [comment]
Hello @i-ron-y,<br /><br/>Thanks again for the PR!<br />                                              <br />We want to show our appreciation by sending                                              you some swag. Could you please fill out                                              <a href="https://dx.sendgrid.com/swag">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

File: .htmllintrc [hunk 1]
```diff
@@ -2,6 +2,6 @@
     // names of npm modules to load into htmllint
     "plugins": [],
 
-    "attr-bans": "none",
+    "attr-bans": false,
```
Comment (mbernier) [inline .htmllintrc]: @two24studios - just a heads up `none` was throwing errors in an html linter, but `false` passes.

**Pull request merged**